packages/nvidia_nat_core/src/nat/front_ends/fastapi/routes/async_generation.py

```
@@ -0,0 +1,174 @@
+from nat.runtime.session import SessionManager
+
+logger = logging.getLogger(__name__)
+
+
+def _job_status_to_response(worker: Any, job):
+    job_output = job.output
+    if job_output is not None:
+        try:
+            job_output = json.loads(job_output)
+        except json.JSONDecodeError:
+            logger.error("Failed to parse job output as JSON: %s", job_output)
```

Ruff (0.15.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)